src/Tools/AssemblyRenamerCLI/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Mono.Cecil;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AssemblyRenamerCLI
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var executingPath = Assembly.GetExecutingAssembly().Location;
+            var xamlBehaviorsPath = Path.Combine(Path.GetDirectoryName(executingPath), "Microsoft.Xaml.Behaviors.dll");
```

Comment:
Can we make this tool generic by supplying dll and namespace names as command line args?

Reply:
sure, that is definitely possible, will look into it.